gcn_email/core.py

```diff
@@ -106,7 +106,7 @@ def kafka_message_to_email(message):
             maintype="application",
             subtype="xml",
         )
-    elif topic.startswith("gcn.notices."):
+    elif topic.startswith("gcn.notices.") or topic.startswith("igwn."):
```

Comment:
Note that you need to subscribe to the topic as well in `subscribe_to_topics`.

Reply:
🤦‍♂️ good catch, added in latest commit